examples/server/v1/components.py

```diff
@@ -325,15 +325,13 @@ async def get_geometries(self, extra: Optional[Dict[str, Any]] = None, **kwargs)
 class ExampleCamera(Camera):
     def __init__(self, name: str):
         p = Path(__file__)
-        img = Image.open(p.parent.absolute().joinpath("viam.webp"))
+        img = Image.open(p.parent.absolute().joinpath("viam.jpeg"))
```

Comment:
Ran into some issues with webp, didn't feel like debugging since we don't use pillow internally anymore